src/install.js

```diff
@@ -218,10 +218,10 @@ Run ppm -v after installing Git to see what version has been detected.\
         if (body == null) { body = {}; }
         if (error != null) {
           message = `Request for package information failed: ${error.message}`;
-          if (error.code) { message += ` (${error.code})`; }
+          if (error.status) { message += ` (${error.code})`; }
```

Comment:
It's been a while since I looked at this code. But I think your suggestion is correct, that the `status` needs to be added to the message as well, whereas I only replaced what was causing tests to fail.
Since I don't believe the `code` object exists anymore